king_phisher/client/assistants/campaign.py

```diff
@@ -144,6 +201,35 @@ def is_editing_campaign(self):
 	def is_new_campaign(self):
 		return self.campaign_id is None
 
+	@property
+	def _url_thread_is_ready(self):
+		return self._url_thread is None or not self._url_thread.is_alive()
+
+	@property
+	def _server_uses_ssl(self):
+		return any(address['ssl'] for address in self.config['server_config']['server.addresses'])
+
+	def signal_expander_activate(self, expander):
+		return
+
+	def signal_kpm_import_clicked(self, _):
```

Comment:
Instead of a giant button for this, can you put it on it's own tab in the assistant? Just a simple tab right after the URL selection with an entry and a spot to open the file selection dialog like what exists in the Mailer Configuration tab.